packages/taco/schema-docs/condition-schemas.md

```diff
@@ -0,0 +1,228 @@
+
+## PlainString
+
+Any string that is not a Context Parameter i.e. does not start with `:`.
+
+_String._
+
+## ContextParam
+
+A Context Parameter i.e. a placeholder used within conditions and specified at the encryption time, whose value is provided at decryption time.
+
+_String which matches the regular expression `/^:[a-zA-Z_][a-zA-Z0-9_]*$/`._
+
+## ParamOrContextParam
+
+_Union of the following possible types:_
+
+- [PlainString](#plainstring), `boolean` _or_ `number`
+- [ContextParam](#contextparam)
+- undefined
+
+## CompoundCondition
+
+_Object containing the following properties:_
+
+| Property            | Type                                                                     | Default      |
+| :------------------ | :----------------------------------------------------------------------- | :----------- |
+| `conditionType`     | `'compound'`                                                             | `'compound'` |
+| **`operator`** (\*) | `'and' \| 'or' \| 'not'`                                                 |              |
+| **`operands`** (\*) | _Array of at least 1  and  at most 5 [AnyCondition](#anycondition) item_ |              |
+
+_(\*) Required._
+
+## ContractCondition
+
+_Object containing the following properties:_
+
+| Property                   | Type                                                         | Default      |
+| :------------------------- | :----------------------------------------------------------- | :----------- |
+| `conditionType`            | `'contract'`                                                 | `'contract'` |
+| **`chain`** (\*)           | `number` (_int, ≥0_)                                         |              |
+| **`method`** (\*)          | `string`                                                     |              |
+| **`parameters`** (\*)      | _Array of [ParamOrContextParam](#paramorcontextparam) items_ |              |
+| **`returnValueTest`** (\*) | [ReturnValueTest](#returnvaluetest)                          |              |
+| **`contractAddress`** (\*) | `string`                                                     |              |
+| `standardContractType`     | `'ERC20' \| 'ERC721'`                                        |              |
+| `functionAbi`              | [FunctionAbi](#functionabi)                                  |              |
+
+_(\*) Required._
+
+## FunctionAbi
+
+_Object containing the following properties:_
+
+| Property                   | Type                                                                    |
+| :------------------------- | :---------------------------------------------------------------------- |
+| **`name`** (\*)            | `string`                                                                |
+| **`type`** (\*)            | `'function'`                                                            |
+| **`inputs`** (\*)          | _Array of at least 0 [FunctionAbiVariable](#functionabivariable) item_  |
```

Comment:
Whatever is generating the information for arrays, can it use `"item(s)"`? That way we don't have to worry about singular/plural contexts when it is used.

Reply:
Done!